src/components/Appbar.js

```diff
@@ -0,0 +1,139 @@
+        marginLeft: drawerWidth,
+        width: `calc(100% - ${drawerWidth}px)`,
+        transition: theme.transitions.create(['width', 'margin'], {
+            easing: theme.transitions.easing.sharp,
+            duration: theme.transitions.duration.enteringScreen,
+        }),
+    },
+    menuButton: {
+        marginRight: 36,
+    },
+    hide: {
+        display: 'none',
+    },
+    drawer: {
+        width: drawerWidth,
+        flexShrink: 0,
+        whiteSpace: 'nowrap',
+    },
+    drawerOpen: {
+        width: drawerWidth,
+        transition: theme.transitions.create('width', {
+            easing: theme.transitions.easing.sharp,
+            duration: theme.transitions.duration.enteringScreen,
+        }),
+    },
+    drawerClose: {
+        transition: theme.transitions.create('width', {
+            easing: theme.transitions.easing.sharp,
+            duration: theme.transitions.duration.leavingScreen,
+        }),
+        overflowX: 'hidden',
+        width: theme.spacing(7) + 1,
+        [theme.breakpoints.up('sm')]: {
+            width: theme.spacing(9) + 1,
+        },
+    },
+    toolbar: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'flex-end',
+        padding: theme.spacing(0, 1),
+        // necessary for content to be below app bar
+        ...theme.mixins.toolbar,
+    },
+    content: {
+        flexGrow: 1,
+        padding: theme.spacing(3),
+    }
+});
+
+
+class Appbar extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { open: false }
+    }
+
+    handleDrawerOpen = () => {
+        this.props.toggleSidebar();
+    };
+
+
+    render() {
+
+        const { classes, theme } = this.props;
+
+        return (
+            <AppBar
+                position="fixed"
+                className={clsx(classes.appBar, {
+                    [classes.appBarShift]: this.props.open,
+                })}
+            >
+                <Toolbar>
+                    <IconButton
+                        color="inherit"
+                        aria-label="open drawer"
+                        onClick={this.handleDrawerOpen}
+                        edge="start"
+                        className={clsx(classes.menuButton, {
+                            [classes.hide]: this.props.open,
+                        })}
+                    >
+                        <MenuIcon />
+                    </IconButton>
+                    <Typography variant="h6" noWrap>
+                        MySpot
+          </Typography>
+                    <Button variant="h6" color="inherit">
+                        Profile
+          </Button>
+                    <Button variant="h6" color="inherit">
+                        Feed
+                    </Button>
+                    <Button variant="h6" color="inherit">Login</Button>
+                </Toolbar>
+            </AppBar>
+
+
+        )
+    }
+
+}
+
+const mapStateToProps = state => {
```

Comment:
if you wrap the curlies in brackets you can avoid the return statement for example.
const mapStateToProps = (state) => ({
    somestate: value
});
just a suggestion :D